lib/pbench/server/__init__.py

```diff
@@ -17,9 +17,9 @@
 # with Python syntax.
 JSONSTRING = str
 JSONNUMBER = Union[int, float]
-JSONVALUE = Union["JSONOBJECT", "JSONARRAY", JSONSTRING, JSONNUMBER, bool, None]
-JSONARRAY = List[JSONVALUE]
-JSONOBJECT = Dict[JSONSTRING, JSONVALUE]
+JSONARRAY = List["JSONVALUE"]
+JSONOBJECT = Dict[JSONSTRING, "JSONVALUE"]
+JSONVALUE = Union[JSONOBJECT, JSONARRAY, JSONSTRING, JSONNUMBER, bool, None]
```

Comment:
I don't know about your linter, Dave, but mine cannot seem to cope with recursive type definitions.  (I suppose I don't blame it -- they are a little bit tricky to enumerate...I suppose I should be happy that this definition doesn't crash it!)  However, by putting `JSONVALUE` last, there are fewer "quoted" references, and `JSONVALUE` actually ends up with a `dict` in the list at least some of the time.

(Hopefully, this doesn't break anything....)

Reply:
> trading two quoted references for two different quoted references makes Pycharm happier?

I don't know about "happier" per se, but it does improve the results ever so slightly (but, sadly, insufficiently to actually address the problem that I was trying to fix...but, I still think it's worth the change).

> the List should become list and the Dict should become dict.

Done.  (I'll push it in a moment...there might be something else to tweak.)